Lib/test/test_capi/test_complex.py

```diff
@@ -0,0 +1,154 @@
+import unittest
+import warnings
+
+from test.test_capi.test_getargs import (BadComplex, BadComplex2, Complex,
+                                         FloatSubclass, Float, BadFloat,
+                                         BadFloat2, ComplexSubclass)
+from test.support import import_helper
+
+
+_testcapi = import_helper.import_module('_testcapi')
+
+NULL = None
+
+class BadComplex3:
+    def __complex__(self):
+        raise RuntimeError
+
+
+class CAPIComplexTest(unittest.TestCase):
+    def test_check(self):
+        # Test PyComplex_Check()
+        check = _testcapi.complex_check
+
+        self.assertTrue(check(1+2j))
+        self.assertTrue(check(ComplexSubclass(1+2j)))
+        self.assertFalse(check(Complex()))
+        self.assertFalse(check(3))
+        self.assertFalse(check([]))
```

Comment:
Instead of the test for list (did you copy this from `test_bytes`?), add a test for float.
```suggestion
        self.assertFalse(check(3.0))
```